warehouse/accumulo-extensions/src/main/java/datawave/ingest/table/balancer/ShardRendezvousHostBalancer.java

```diff
@@ -0,0 +1,226 @@
+                    log.error("Unable to parse {} value ({}}) as an integer.  Defaulting to {}", SHARDED_MAX_MIGRATIONS, maxMigrationsProp, maxMigrations);
+                }
+            }
+        } catch (Exception e) {
+            log.warn("Failed to get {}}.  Defaulting to {}", SHARDED_MAX_MIGRATIONS, maxMigrations, e);
+        }
+        return maxMigrations;
+    }
+
+    private List<Map.Entry<Long,Matcher>> getTiers(Configuration tableConfig) {
+        List<Map.Entry<Long,Matcher>> configuredTiers = new ArrayList<>();
+        Set<String> tiers = ShardedTableDateBasedTieredVolumeChooser.listTiers(tableIdToBalance, tableConfig);
+        for (String tier : tiers) {
+            long daysBack = ShardedTableDateBasedTieredVolumeChooser.getTierDaysBack(tableIdToBalance, tableConfig, tier);
+            String regex = ShardedTableDateBasedTieredVolumeChooser.getTierProperty(tableIdToBalance, tableConfig, tier, TSERVERS_PREFIX);
+            var pattern = Pattern.compile(regex);
+            configuredTiers.add(new AbstractMap.SimpleImmutableEntry<>(daysBack, pattern.matcher("")));
+        }
+        return configuredTiers;
+    }
+
+    protected List<Long> getDaysBack(List<Map.Entry<Long,Matcher>> configuredTiers, TabletServerId tabletServerId) {
+        ArrayList<Long> db = new ArrayList<>();
+        for (var entry : configuredTiers) {
+            long daysBack = entry.getKey();
+            Matcher matcher = entry.getValue();
+            matcher.reset(tabletServerId.getHost() + ":" + tabletServerId.getPort());
+            if (matcher.matches()) {
+                db.add(daysBack);
+            }
+        }
+        return db;
+    }
+
+    protected LocalDate today() {
+        return LocalDate.now();
+    }
+
+    public static LocalDate parse(String date) {
+        try {
+            return LocalDate.parse(date, FORMATTER);
+        } catch (DateTimeParseException e) {
+            log.warn("Shard: {} is not formatted correctly", date);
+            return LocalDate.EPOCH;
+        }
+
+    }
+
+    private static final String DATE_PATTERN = "yyyyMMdd";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern(DATE_PATTERN);
+
+    @Override
+    protected Function<String,Map<String,List<TabletServerId>>> getServerPartitioner(Collection<TabletServerId> allTservers) {
+
+        final List<Map.Entry<Long,Matcher>> configuredTiers = getTiers(tableConfig);
+
+        if (configuredTiers.isEmpty()) {
+            throw new IllegalStateException("Tier configuration is not set");
+        }
+
+        TreeMap<Long,Map<String,List<TabletServerId>>> serverPartitioningMap = new TreeMap<>();
+
+        allTservers.forEach(tabletServerId -> {
+            var daysBack = getDaysBack(configuredTiers, tabletServerId);
+            if (!daysBack.isEmpty()) {
+                for (var db : daysBack) {
+                    log.trace(" daysBack:{} tserver:{}", daysBack, tabletServerId);
```

Comment:
Do you want to log each day vs the group of days?
```suggestion
                    log.trace(" daysBack:{} tserver:{}", db, tabletServerId);
```